plugins/couchbase/couchbaseembed.cpp

```diff
@@ -63,6 +63,31 @@ namespace couchbaseembed
     // Plugin Classes
     //--------------------------------------------------------------------------
 
+    void reportIfQueryFailure(Couchbase::Query * query)
+    {
+        auto status = query->meta().status();
+        if (status.errcode())
+        {
+            if (status.isNetworkError())
```

Comment:
I thought about that, but realized we really didn't gain anything in terms of performance, and introduces a variable of little use.